napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -134,12 +135,12 @@ def __init__(self, layer):
             self.changePlaneThickness
         )
 
-        sld = QSlider(Qt.Orientation.Horizontal, parent=self)
+        sld = QDoubleSlider(Qt.Orientation.Horizontal, parent=self)
         sld.setFocusPolicy(Qt.FocusPolicy.NoFocus)
-        sld.setMinimum(0)
-        sld.setMaximum(100)
-        sld.setSingleStep(1)
-        sld.setValue(int(self.layer.iso_threshold * 100))
+        cmin, cmax = self.layer.contrast_limits
+        sld.setMinimum(cmin)
+        sld.setMaximum(cmax)
+        sld.setValue(cmin + (cmax - cmin) / 2)
```

Comment:
Ahh, thanks. At first this was awkward because on my dataset the full range is much bigger than the actual values, but I discovered the mechanism to reset the range and I agree this makes more sense.